enterprise/frontend/src/embedding-sdk/cli/run.ts

```diff
@@ -44,6 +43,7 @@ export const CLI_STEPS = [
     id: "generateReactComponentFiles",
     executeStep: generateReactComponentFiles,
   },
+  { id: "setupLicense", executeStep: setupLicense },
```

Comment:
We setup the Metabase license at the very end of the process. Defer the sign-up process and let the user feel a sense of success first 😄